lib/generators/factory_bot/model/model_generator.rb

```diff
@@ -78,7 +78,7 @@ def filename
         if factory_bot_options[:filename_proc].present?
           factory_bot_options[:filename_proc].call(table_name)
         else
-          [table_name, filename_suffix].compact.join('_')
+          [table_name, filename_suffix].compact.join(" ")
```

Comment:
Was this supposed to remain an underscore?

Reply:
@mikegee good catch, I fixed this [here](https://github.com/thoughtbot/factory_bot_rails/pull/299/commits/41330d140cd0b22ffa7e1de7ae864277dfb530cd)